controller.yml

```diff
@@ -44,3 +44,26 @@
      - python-zmq
      - apache2
   - include: controller_extension.yml
+
+- hosts: cc*
```

Comment:
would it be possible to separate out this file? It would be great as it will help in applying atomic scripts.